packages/slice-machine/src/modules/slices/index.ts

```diff
@@ -243,19 +261,10 @@ export function* createSliceSaga({
 export function* renameSliceSaga({
   payload,
 }: ReturnType<typeof renameSliceCreator.request>) {
+  const { libName, sliceId, newSliceName } = payload;
   try {
-    yield call(
-      renameSlice,
-      payload.sliceId,
-      payload.newSliceName,
-      payload.libName
-    );
-    //TODO: avoid refetch of state. Either manually update all the changes within the reducer,
-    //or make the renameSlice api call return the new libraries, so that it's output can be used.
-    const { data: serverState } = (yield call(getState)) as SagaReturnType<
```

Comment:
so this is where the reload happened?

Reply:
this is where it used to happen, but now we removed it and just do it using redux :)